android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -510,6 +516,10 @@ public void dispatchDownloadProgressEvent() {
                         e.printStackTrace();
                         saveFailedUpdate(updatePackage);
                         promise.reject(e);
+                    } catch (CodePushMalformedDataException e) {
```

Comment:
this was to fix a bug discovered during testing that threw this error but was never caught and crashed the app on startup